apps/src/storage/dataBrowser/dataVisualizer/VisualizerModal.jsx

```diff
@@ -93,11 +111,10 @@ class VisualizerModal extends React.Component {
           fullHeight
         >
           <div style={{overflow: 'auto', maxHeight: '90%'}}>
-            <h1> Explore {this.props.tableName} </h1>
-            <h2> Overview </h2>
+            <h2> Explore {this.props.tableName} </h2>
```

Comment:
i18n

Reply:
Nothing for datasets has been i18n-ized yet. Planning on doing that once things are a little more finalized, since a lot of these strings have changed a couple times. Definitely will be done before this comes out of experiment.